### ISSUE TITLE
Running metacell_import generates an empty std/ directory.

### ISSUE THREAD

**Issue by pburnham50** [tanaylab/metacell.py#1]
Hi,

Congrats on the paper; the concept is very cool. I was interested in running Metacell to look at gene modules using correlation. Here I am using one of the Tabula Muris datasets which I reverted back to fastq and aligned via kallisto bustools.

I can't seem to get past the first step though. When I run "metacell_import" it creates an empty std directory. Just wanted to check that I have everything set up correctly.

structure:
root/sample/raw/

in raw/ there are three files: barcodes.tsv, genes.tsv, matrix.mtx

genes.tsv contains Ensembl IDs:
"ENSMUSG00000000001
ENSMUSG00000000003
ENSMUSG00000000028
..."

barcodes.tsv contains cell identifier:
"TM_Lung_AAACCTGAGACAAAGG
TM_Lung_AAACCTGAGACTAAGT
TM_Lung_AAACCTGAGAGCCCAA
..."

matrix.mtx is the sparse matrix format of raw counts:
"%%MatrixMarket matrix coordinate integer general
55422 3764 8138756
11 1 2
12 1 1
..."

Can  you please advise?

**Comment by orenbenkiki** [comment]
Keeping in mind this is alpha-level software and will break as soon as you breathe the wrong way... I'm actively tweaking it (e.g., we are now working on improving the outlier detection scheme, which is currently sub-optimal). I would definitely not trust the outputs "too much" until we actually publish and release a more stable version. That said, it seems to be working reasonably well.

I'm not in front of a computer right now, but from memory, I think that the raw directory should contain one sub-directory per "batch" (since we typically deal with large data sets that comprise of multiple such batches). Try something like creating `raw/all` and moving the files into it.

**Comment by pburnham50** [comment]
Thanks, unfortunately that did not work but I understand that it's an alpha release. Was trying to use the original R version but my R is having trouble compiling the tgstat package at install.

**Comment by orenbenkiki** [comment]
Yes, my bad, I should have sent you to the R package in the 1st place, it is more robust for now.
However, I'm not the right guy to ask about it - try opening an issue in the R package repository: https://github.com/tanaylab/metacell
